src/Components/ProgramContainer/index.jsx

```diff
@@ -1,15 +1,18 @@
 import './ProgramContainer.css'
 import Draggable from "react-draggable";
+import AboutPage from "../AboutPage/index.jsx";
 
 const ProgramContainer = ({toggleProgram, text, children}) => {
     return (
         <Draggable handle="header">
             <div className="program resize overflow-auto">
-                <header className="header h-16">
-                    <span className="text-2xl">Program</span>
+                <header className="header h-16 bg-green-700 text-center">
+                    <span className="text-3xl">{text}</span>
                     <span className="close-btn absolute top-2 right-2" onClick={() => toggleProgram(false)}>Close</span>
```

Comment:
<img width="1289" alt="image" src="https://github.com/iO-Academy/23-jul-OS-FE/assets/137928354/b7371ffb-8b2f-4880-9f7e-376d9b30f08a">

- Heading doesn't fit nicely in the header bar
- close button meant to be an icon button as per the wireframe?
- program container to have rounded corners as per the wireframe?